android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java

```diff
@@ -358,16 +358,23 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 return true;
 
             case R.id.action_share:
+                StringBuilder speakersNameBuilder  = new StringBuilder();
+                for(Speaker speaker : session.getSpeakers()){
+                    speakersNameBuilder.append(speaker.getName() + ", ");
+                }
+                //deleting comma and space after last name
+                speakersNameBuilder.delete(speakersNameBuilder.length() - 2, speakersNameBuilder.length() - 1);
```

Comment:
Even though i asked you to add that comment, now i feel it would be much better to use a boolean flag to add a comma and space i.e. don't add a comma and space when it's not required rather that adding it first and then deleting it.
Right now, the problem is, let's say the session.getSpeakers() returns an empty list. Then that for loop won't run at all but still you will try to delete something that isn't there and that would lead to error.

Can you use a boolean flag and only add comma and space when required ?

Reply:
@srv-twry The list will always have a speaker right!

Reply:
I hope it has, but always good to be safe.

Reply:
@srv-twry Well I could put a null check ?

Reply:
@srv-twry Or I could implement your suggested method.

Reply:
The problem won't be resolved with a null check. Here is the problem:
1. You created a speakerNameBuilder object with empty string.
2. Now the session.getSpeakers() returns an empty list.
3. You try to delete the last two characters of the speakerNameBuilder, but wait !, it's empty. So you will get the error. It's like you're trying to delete something which isn't there.

Got it ?

Reply:
@srv-twry Yeah I did the changes 😅